.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+
+## Our Git Work Flow
+
+* for smaller projects we're essentially following the feature branch workflow that can be found [here](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+* For larger projects we're following the process discused below, which is better suited to ongoing projects with multiple releases and CI, you can find a detailed discussion [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)
+
+We use two primary branches.  `master` and `develop`.  `develop` is what you'll create pull requests against, and should be deployed to staging environments for testing. Once enough work has gone in to create a  deliverable & deployable piece of value, and all testing is complete we create a release branch (this is where we start adding version numbers, add documentation and do any prep related to the release), this then gets merged into master, which will then be deployed to a production environment,  we should merge `master` back into `development`.  Ideally `master` should always represent what is in production. 
+
+
+### Contributing to the project
+Always branch, and use the below naming schema for branches. Once you've completed the work for that branch, push it up to github and create a Pull Request against development.
+
+
+#### Naming Branches
+We prefix our branch names to make it easier to see what branches relate to.
+
+* `feature/<name>`  - a new feature
+* `fix/<name>` - fixes an issue
+* `style/<name>` - small front end updates
+* `hotfilx/<name>` - fixes against a release
+* `release/<name>` - preparing for a release
+
+
+### Pull Requests
+Pull requests should be kept as small as possible, this eases review, and avoids unnessesary merge confilcts.  Pull Requests should be reviewed by two team members.
+
+**NB :** for front-end changes and style fixes, adding screen shots to the pull request is tremendously helpful in explaining what was done and why.
+
+
+### Commits & Commit Messages
+Commits should also be kept as small as possible, commit titles should be less than 80 characters, and additional descriptions should be used when neccessary.  
```

Comment:
Agree more info is useful, seems pretty standard that the title should be 80 char, again viewing in the terminal, on Github, in tower, keeps it concise and easy.  Don't think anyone will have a tantrum if we go over here and there, but longer than that probably should be a description @SkippyZA @JamesGarethSmith  ?